config/locales/en/providers.yml

```diff
@@ -466,12 +466,32 @@ en:
           organisation: An organisation
       opponent_individuals:
         show: &new
-          page_title: Opponent's name
+          page_title: Opponent
         new: *new
         form:
           first_name: First name
           last_name: Last name
-      opponent_organisations:
+      opponent_existing_organisations:
+        create:
+          search_and_select: Search for and select an organisation
+        index:
+          clear_search: Clear search
+          enter_manually: enter manually
+          enter_manually_html: If you can not find the organisation %{href}
+          heading_1: Opponent
+          heading_2: Search for an organisation
+          no_results: No results found.
+          search_help_example: Enter 3 or more letters to start searching. Select one organisation at a time. You can add more later.
+          accessibility_message: Tab out of this box to select returned options
```

Comment:
This message was raised in the accessibility review for needing clarification too, does this already reflect that?